Functions/Mock.ps1

```diff
@@ -1249,16 +1282,15 @@ function Get-DynamicParametersForCmdlet {
         return
     }
 
+    if ($null -eq $InputArgs -or $InputArgs.Count -eq 0) {
```

Comment:
if the `InputsArgs` is null or empty there is no point on trying to find dynamic parameters since the command invocation didn't specify them